### PR TITLE
Added support for actionSanitiser and stateSanitiser in @ngrx/store-devtools

### PR DESCRIPTION
**Pull request by roopkt**
This is the fix to pass state and action sanitizer to redux dev tool extensions for #494.

### CODE REVIEW

**Comment by brandonroberts** [comment]
@roopkt please fix merge conflicts and we'll land this PR.

**Comment by brandonroberts** [comment]
@roopkt merge conflicts still remain. See the details from the failed tests

**Comment by roopkt** [comment]
@brandonroberts  I am still working on the merge.  I have question when are we planing to review #524 pull request.

**Comment by roopkt** [comment]
Hey @brandonroberts 
Finally merge happened successfully.

File: modules/store-devtools/spec/config.spec.ts [hunk 1]
```diff
@@ -7,4 +7,21 @@ describe('StoreDevtoolsOptions', () => {
...
+
+  it('can be initialized with actionSanitizer', () => {
+    const options = new StoreDevtoolsConfig();
+    function sanitizer(action: Action, id: number): Action {
+      return action;
+    }
+    options.actionSanitizer = sanitizer;
+    expect(options.actionSanitizer).toEqual(sanitizer);
+  });
+  it('can be initialized with stateSanitizer', () => {
```
Comment (brandonroberts) [inline modules/store-devtools/spec/config.spec.ts]: Add a line in between these tests

File: modules/store-devtools/spec/config.spec.ts [hunk 2]
```diff
@@ -7,4 +7,21 @@ describe('StoreDevtoolsOptions', () => {
...
+    const options = new StoreDevtoolsConfig();
+    function stateSanitizer(state: any, index: number): any {
+      return state;
+    }
+    options.actionSanitizer = stateSanitizer;
```
Comment (brandonroberts) [inline modules/store-devtools/spec/config.spec.ts]: Shouldn't this be `stateSanitizer`?

File: modules/store-devtools/spec/extension.spec.ts [hunk 3]
```diff
@@ -52,16 +47,28 @@ describe('DevtoolsExtension', () => {
         'ngrx-store-1509655064369'
       );
     });
+
+    function myActionSanitizer() {
+      return { type: 'sanitizer' };
+    }
+    function myStateSanitizer() {
```
Comment (brandonroberts) [inline modules/store-devtools/spec/extension.spec.ts]: Add a line in between these functions

Comment (roopkt) [reply modules/store-devtools/spec/extension.spec.ts]: sure

File: modules/store-devtools/spec/config.spec.ts (hunk 2 above)
Comment (roopkt) [reply modules/store-devtools/spec/config.spec.ts]: yes

File: modules/store-devtools/spec/config.spec.ts (hunk 1 above)
Comment (roopkt) [reply modules/store-devtools/spec/config.spec.ts]: sure

**Comment by aboyton** [comment]
Thanks for doing (and merging) this. Are you planning on a patch release anytime soon with this?

**Pull request merged**